doc/14_data_frames.md

```diff
@@ -42,20 +42,18 @@ Example Changing Cluster/Keyspace Level Properties
 val conf = new SparkConf()
   .set("ClusterOne/spark.cassandra.input.split.size_in_mb","32")
```

Comment:
I think it would be nice to have a shortcut for this, I don't like having to remember the syntax here
Cluster:keyspace/key